libs/server/Resp/Bitmap/BitmapCommands.cs

```diff
@@ -113,39 +113,30 @@ internal sealed unsafe partial class RespServerSession : ServerSessionBase
         /// The bit is either set or cleared depending on value, which can be either 0 or 1.
         /// When key does not exist, a new key is created.The key is grown to make sure it can hold a bit at offset.
         /// </summary>
-        private bool NetworkStringSetBit<TGarnetApi>(byte* ptr, ref TGarnetApi storageApi)
+        private bool NetworkStringSetBit<TGarnetApi>(ref TGarnetApi storageApi)
```

Comment:
Not for this PR, but wanted to highlight that none of these methods need to be `bool` any longer, correct?

Reply:
That's correct

Reply:
OK, let us add this to a future cleanup.